src/frontend/src/components/GitHubContributorCard/GitHubContributorCard.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { makeStyles, useTheme } from '@material-ui/core/styles';
+import { Card, CardContent, Typography, Avatar, Grid } from '@material-ui/core';
+import useTelescopeContributor from '../../hooks/use-telescope-contributor';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: theme.palette.primary.light,
+  },
+  typography: {
+    fontSize: 13,
+  },
+  links: {
+    color: theme.palette.text.secondary,
+    textDecorationLine: 'none',
+    '&:hover': {
+      textDecorationLine: 'underline',
+    },
+    alignItems: 'flex-start',
+  },
+}));
+
+const GitHubContributorCard = () => {
+  const classes = useStyles();
+  const theme = useTheme();
+  const { contributor, error } = useTelescopeContributor();
+
+  if (contributor !== undefined && contributor !== null) {
```

Comment:
This is an anti-pattern.  Instead of deeply indenting the majority of a function's code, you should invert your logic and deal with the negative case first, return, and then continue with the normal case without an indent.  In your case:

```js
if(!contributor) {
  return null;
}

return ( /* rest of your code here */);
```